arachni/app/tasks.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+from celery import Celery
+
+celery = Celery('arachni')
+celery.config_from_object('celeryconfig')
+
+def call_scanner(args):
+    cmd = ' '.join(args)
+    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    stdout, stderr = p.communicate()
+    if stdout:
+        stdout = stdout.replace('\n', '<br />')
```

Comment:
- Even if it was not a security issue, you're mixing data and presentation layers
- This is arbitrary data, we have absolutely no idea what this process may or may not spit out (especially since there's a lingering shell injection vulnerability)
- You are already installing `MarkupSafe` via requirements.txt, doing this right is as simple as one call to `markupsafe.escape`